src/vs/workbench/contrib/testing/common/testItemCollection.ts

```diff
@@ -673,7 +673,7 @@ export const createTestItemChildren = <T extends ITestItemLike>(api: ITestItemAp
 
 			for (const item of items) {
 				if (!(item instanceof checkCtor)) {
-					throw new InvalidTestItemError(item.id);
+					throw new InvalidTestItemError((item as ITestItemLike).id);
```

Comment:
Curious what the errors where here... I'll look into that

Reply:
Thanks. I think the type signature may be wrong here. Right now `checkCtor: Function` but it should likely be a more specific type instead so that TS doesn't think `instanceof` check is pointless